extra/utils/Validation.js

```diff
@@ -1,24 +1,11 @@
-let users = [
-    {
-        traineeEmail : 'shashank.baranawal@successive.tech',
-        reviewerEmail : 'naman.parashar@successive.tech',
-    },
-    {
-        traineeEmail : 'shashank.baranwal@gmail.com',
-        reviewerEmail : 'naman.parashar@gmail.com'
-    }
-]
-
-function validateEmail(email){
-    const regix = /\w+.\w+@successive.tech$/i;
-    return regix.test(email)
-}
-
+import {user} from '../constant'
```

Comment:
unused import

Reply:
I checked all the files  then I found by mistaken i have wrongly imported .